src/main/java/mate/academy/spring/config/AppConfig.java

```diff
@@ -44,4 +46,9 @@ public LocalSessionFactoryBean getSessionFactory() {
         factoryBean.setPackagesToScan("mate.academy.spring.model");
         return factoryBean;
     }
+
+    @Bean
+    public PasswordEncoder getEncoder() {
+        return new BCryptPasswordEncoder();
```

Comment:
Let's set strength and SecurityRandom in PasswordEncoder constructor. You can use 10 strength and just 
new SecurityRandom() by default.

Reply:
https://docs.spring.io/spring-security/site/docs/current/api/org/springframework/security/crypto/bcrypt/BCryptPasswordEncoder.html

default value of strength is 10 
I google example and don't see that sombody sets this value